news/2023/2023-09-16-bn-mentorship-cycle-1.md

```diff
@@ -0,0 +1,40 @@
+
+After extensive feedback gathered from the test phase, and as said in the title above, we're proud to open sign-ups for the first official cycle of the BN Mentorship Program!
+
+But wait, you ask… What the *Haitch E. Double Hockey Sticks* is this "BN Mentorship Program"?
+
+It's a modding course ran by the [Nomination Assessment Team](/wiki/People/Nomination_Assessment_Team) (us, the authors of this post) designed to get a modder's knowledge and skill up to what we consider to be the standard skill level of a BN. Partaking in the course will have you learn how to use the essential software, the big ranking rules you need to get acquainted with, as well as lots, and we mean *lots* of modding sessions to forge your skills in practice.
+
+Unlike the prototype cycle, the mentors will be picking the mentees this time around. We'll take into consideration your previous BN applications (if you've had any), your recent mods and activity, if you have the required 200 *[kudosu!](/wiki/Modding/Kudosu)* to be a candidate, as well as your general *vibe*. Don't be discouraged if you don't get picked, as this is built to be a cyclical thing, which means there's always the next cycle to look forward to.
+
+To clarify, this program does not replace a regular application. You can still join in the old-fashioned way, this is just an alternative for people who are interested in improving or have some trouble with any particular topic when applying, and often feel stuck in the current system — struggling to move onto the next level.
+
+---
+
+This cycle is expected to last from **September 30th** to **December 19th**, and we'll start taking sign-ups… right now! (assuming you aren't some fancy future human-lizard hybrid reading this in the future).
+
+The available game modes for this cycle are **osu!**, **osu!taiko**, and **osu!mania**. Here's the list of the available mentors for each mode:
+
+| ![osu!](/wiki/shared/mode/osu.png "osu!") Mentor | ![osu!taiko](/wiki/shared/mode/taiko.png "osu!taiko") Mentor | ![osu!mania](/wiki/shared/mode/mania.png "osu!mania") Mentor |
+| :-- | :-- | :-- |
+| ::{ flag=BR }:: [Dada](https://osu.ppy.sh/users/9119507) | ::{ flag=TN }:: [Hivie](https://osu.ppy.sh/users/14102976) | ::{ flag=ID }:: [Maxus](https://osu.ppy.sh/users/4335785) |
+| ::{ flag=BE }:: [enneya](https://osu.ppy.sh/users/10959501) | ::{ flag=US }:: [radar](https://osu.ppy.sh/users/7131099) | ::{ flag=ES }:: [RandomeLoL](https://osu.ppy.sh/users/7080063) |
+| ::{ flag=DE }:: [FuJu](https://osu.ppy.sh/users/10773882) |  |  |
+| ::{ flag=BE }:: [yaspo](https://osu.ppy.sh/users/4945926) |  |  |
+
+Sign-ups will be open until **September 26th**, and you can sign up **[>>>>>HERE<<<<<](https://forms.gle/YCPNaGSMEXmyDtvKA)**. Please read the form carefully, and only sign up if you're available and willing to attend and commit to classes for the duration of the course.
```

Comment:
planning to use a mini banner for the sign up form, will update when i get the asset

> Please read the form carefully, and only sign up if you're available and willing to attend and commit to classes for the duration of the course.

honestly we'd prefer to keep this, because last cycle (and from experience in community mentorship program), it was not rare for people to sign up and do nothing/bare minimum

Reply:
> > Please read the form carefully, and only sign up if you're available and willing to attend and commit to classes for the duration of the course.
> 
> honestly we'd prefer to keep this, because last cycle (and from experience in community mentorship program), it was not rare for people to sign up and do nothing/bare minimum

i left that part as is in the suggestion though 🤔

Reply:
i may be blind and fresh out of a nap